mobile/ios/config/test/GoogleService-Info.plist

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>API_KEY</key>
+	<string>AIzaSyC1DcfWnlt0BQMpX7_z3f0QPTgL1Po7zB8</string>
```

Comment:
🤔 Is it safe to make that `api_key` public?

Reply:
answering my own question. yes it is ok to store this `api_key` in a public repo.

> Unlike how API keys are typically used, API keys for Firebase services are not used to control access to backend resources; that can only be done with Firebase Security Rules (to control which users can access resources) and App Check (to control which apps can access resources).
>
>Usually, you need to fastidiously guard API keys (for example, by using a vault service or setting the keys as environment variables); however, API keys for Firebase services are ok to include in code or checked-in config files.

https://firebase.google.com/docs/projects/api-keys